### PR TITLE
Fix random wind and add a basic arrow indicator

### LINKED ISSUE #1: Wind Speed and Direction Increment
The wind speed and direction currently just increment by a random number forever.

### PR DESCRIPTION
**Pull request by andrewclarkson**
This should fix #1. The parameters may still need some tweaking.

### CODE REVIEW

**Comment by kmarekspartz** [comment]
Cool. If we're doing degrees, why not use Ints? I was thinking it would be the range [0,1), but this works, too. We'll need to handle the wrapping around case and a few other tweaks to make it more realistic, but this is a good start.

File: Wind.elm [hunk 1]
```diff
@@ -27,39 +30,52 @@ updateWind = updateWindSpeed << updateWindDirection
 updateWindSpeed : Model -> Model
 updateWindSpeed m =
   let (prob, seed') = generate (float 0 1) m.seed
-      windSpeed' = m.windSpeed + prob
+      windSpeed' = (m.windSpeed * 0.9) + (prob * 0.1)
```
Comment (kmarekspartz) [inline Wind.elm:33]: This smells of code duplication! Something like:

``` Haskell
slowlyChangeBy x d = (x * 0.9) + (d * 0.1) 
```

Comment (andrewclarkson) [inline Wind.elm:33]: Keeping it DRY functional style. :)

Comment (kmarekspartz) [inline Wind.elm:33]: Keeping things DRY is almost easier when state is explicit.

**Pull request merged**